filecoindot_proofs/src/verify.rs

```diff
@@ -0,0 +1,46 @@
+use cid::Cid;
+use crate::errors::Error;
+
+pub fn verify(proof: Vec<Vec<u8>>, cid: &Cid) -> Result<(), Error> {
+    let bytes = cid.to_bytes();
+    match proof.iter().find(|p| **p == bytes) {
+        Some(_) => Ok(()),
+        None => Err(Error::VerificationFailed)
+    }
+}
```

Comment:
this only verifies that the cid is included the proof somewhere,
we need to also check that the root matches, so a verify function would look like `fn verify (root, proof, cid)`